test/baselines/vis_Widget/package.json

```diff
@@ -27,12 +27,10 @@
   },
   "main": "widgets/test-widget.html",
   "files": [
-    "lib/",
```

Comment:
Seems ok, but there is currently a `lib` directory:
https://github.com/ioBroker/create-adapter/tree/master/test/baselines/vis_Widget/lib

It should probably not exist.

Reply:
Hmm, right. I got mislead by the tools.js that must exist (during development).